pkg/harvester-manager/models/harvesterhci.io.management.cluster.js

```diff
@@ -41,6 +41,16 @@ export default class HciCluster extends ProvCluster {
     return false;
   }
 
+  // We do not allow users to edit Harvester clusters from Cluster Management, so we need to re-enable that action here.
+  get _availableActions() {
+    const out = super._availableActions;
+    const edit = out.find((action) => action.action === 'goToEdit');
+
+    edit.enabled = true;
```

Comment:
this overrides rbac, it should be based on if the user has permissions

Reply:
think it's this.canCustomEdit that you need, needs validating with someone who is a customer owner and member